app/policies/lighthouse_policy.rb

```diff
@@ -11,6 +11,12 @@ def direct_deposit_access?
       user.icn.present? && user.participant_id.present?
   end
 
+  def itf_access?
+    # Need to check for first name as Lighthouse will check for it
+    # and throw an error if it's not present
+    user.participant_id.present? && user.ssn.present? && user.first_name && user.last_name
```

Comment:
I did [ask Lighthouse](https://dsva.slack.com/archives/CE6UEJ6BZ/p1718399125333749?thread_ts=1717100090.559599&cid=CE6UEJ6BZ) for more context about why first name is required since [users sometimes do not have separate first and last names](https://dsva.slack.com/archives/CSFV4QTKN/p1716227359020019?thread_ts=1716227266.736879&cid=CSFV4QTKN), but they are still looking into it, so going to keep this as is since current implementation will error without a first name (we've [seen it before](https://dsva.slack.com/archives/CE6UEJ6BZ/p1713559201747879?thread_ts=1713554782.128669&cid=CE6UEJ6BZ) — link to [alert](https://dsva.slack.com/archives/C05UPRR0HK3/p1713559537205939?thread_ts=1713532971.349289&cid=C05UPRR0HK3) in our channel)

Reply:
Opportunity to Educate me :D 

Why are you checking `.present?` on pid and SSN, but not on name? Everything is a `string`

Reply:
😮 good catch! I'll add `present?` for last name, but I think I can leave it off for first name to allow empty strings/spaces to evaluate to true (which was recommended as a [workaround for some DR specific work](https://dsva.slack.com/archives/CKUCFQAHJ/p1716398562677689?thread_ts=1716227820.542319&cid=CKUCFQAHJ)) — not sure how much it would make a difference for Veterans with a `nil` first name on their MPI profile, but it at least [aligns a bit more with the check LH does](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/concerns/claims_api/target_veteran.rb#L85-L88)